junit-platform-console/junit-platform-console.gradle.kts

```diff
@@ -27,6 +25,11 @@ tasks {
 			include("LICENSE-picocli.md")
 			into("META-INF")
 		}
+		doLast {
+			ToolProvider.findFirst("jar").get().run(System.out, System.err, "--update",
+					"--file", archiveFile.get().asFile.absolutePath,
+					"--main-class", "org.junit.platform.console.ConsoleLauncher")
+		}
```

Comment:
❓ The `Main-Class` attribute is already set below. Why is this necessary?

Reply:
The `MANIFEST.MF` entry is only evaluated when running `java` in _executable jar_ mode.

Here we're talking _launching the main class in a module_: `java [options] --module module[/mainclass] [args...]` -- for when you want to omit the `/mainclass` part.

More details at: https://docs.oracle.com/en/java/javase/11/tools/java.html#GUID-3B1CE181-CD30-4178-9602-230B800D4FAE

Reply:
...and there's **no** `mainclass` directive with the `module` specification: https://docs.oracle.com/javase/specs/jls/se11/html/jls-7.html#jls-7.7

Reply:
Thanks for the explanation!